packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -47,6 +55,13 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
         setAdminProperties(_defaultCurrency, _defaultLiveness, 0.5e18);
     }
 
+    /**
+     * @notice Sets the default currency, liveness, and burned bond percentage.
+     * @dev Only callable by the contract owner (UMA governor).
+     * @param _defaultCurrency the default currency to bond asserters in assertTruthWithDefaults.
+     * @param _defaultLiveness the default liveness for assertions in assertTruthWithDefaults.
+     * @param _burnedBondPercentage the percentage of the bond that is burned if the assertion is disputed.
```

Comment:
```suggestion
     * @param _burnedBondPercentage the percentage of the bond that is sent as fee to UMA Store contract on disputes.
```

Reply:
Should we rename `_burnedBondPercentage` to `_disputeFeePercentage` then?